resources/views/people/relationship/_relationship.blade.php

```diff
@@ -1,6 +1,6 @@
 @foreach ($relationships as $relationship)
   <div class="sidebar-box-paragraph">
-    <span class="silver fw3 ba br2 ph1 {{ htmldir() == 'ltr' ? '' : 'fr' }}">{{ $relationship->relationshipType->getLocalizedName(null, false, $relationship->ofContact->gender->name) }}</span>
+    <span class="silver fw3 ba br2 ph1 {{ htmldir() == 'ltr' ? '' : 'fr' }}">{{ $relationship->relationshipType->getLocalizedName(null, false, $relationship->contactIs->gender->name) }}</span>
```

Comment:
Are you sure of this change? Why?

Reply:
Thanks ! I though it fixed something but it was not.